src/solidity/script/Deploy.s.sol

```diff
@@ -86,25 +87,33 @@ contract Deploy is Script {
     OuterCount12Verifier outerCount12Verifier = new OuterCount12Verifier();
     console.log("Outer (12 subproofs) verifier deployed at:", address(outerCount12Verifier));
 
+    console.log("Deploying Outer (13 subproofs) verifier...");
+    OuterCount13Verifier outerCount13Verifier = new OuterCount13Verifier();
+    console.log("Outer (13 subproofs) verifier deployed at:", address(outerCount13Verifier));
+
     console.log("Deploying ZKPassportVerifier...");
     address rootRegistry = vm.envAddress("ROOT_REGISTRY_ADDRESS");
     ZKPassportVerifier zkPassportVerifier = new ZKPassportVerifier(rootRegistry);
     console.log("ZKPassportVerifier deployed at:", address(zkPassportVerifier));
 
-    // Add verifiers to ZKPassportVerifier
-    address[] memory verifierAddresses = new address[](9);
-    verifierAddresses[0] = address(outerCount4Verifier);
-    verifierAddresses[1] = address(outerCount5Verifier);
-    verifierAddresses[2] = address(outerCount6Verifier);
-    verifierAddresses[3] = address(outerCount7Verifier);
-    verifierAddresses[4] = address(outerCount8Verifier);
-    verifierAddresses[5] = address(outerCount9Verifier);
-    verifierAddresses[6] = address(outerCount10Verifier);
-    verifierAddresses[7] = address(outerCount11Verifier);
-    verifierAddresses[8] = address(outerCount12Verifier);
-    console.log("Adding verifiers to ZKPassportVerifier...");
-    zkPassportVerifier.addVerifiers(vkeyHashes, verifierAddresses);
-    console.log("Verifiers added to ZKPassportVerifier");
+    {
```

Comment:
Moving this to a separate scope as it was running in to "memory out of bounds" error